assembl/static2/css/common.scss

```diff
@@ -84,7 +84,7 @@ textarea.placeholder {
 }
 
 .app-content {
-  margin: $navbar-height 0 0;
+  margin: 40px 0 0;
```

Comment:
Fixed, always, mobile ?

Reply:
I don't know why but this grey space between the navbar and the header appears. 40px instead of $navbar-height which is 60px fix the problem. I didn't wanted to change the value of navbar-height because it is used multiple times in the project. 
<img width="1440" alt="Capture d’écran 2019-07-08 à 13 32 01" src="https://user-images.githubusercontent.com/26386718/60807105-08b03a00-a185-11e9-97a4-40aa709999b3.png">

Reply:
Ship it anyway!